docs/chapters/baseplatform/booting-on-target-hardware.rst

```diff
@@ -2,6 +2,21 @@ Booting on target hardware
 ==========================
 .. note:: Login as user ``root``.
 
+ARP
+---
+
+On the ARP there are three options available for booting.
+
+1. USB: Boot from a USB-stick by simply inserting said USB-stick into one of the
+   USB ports on the arp. Make sure that USB is set as boot source in bios/u-boot
+2. SD-card: Put insert the SD-card into the slot labeled "SD card" located next
```

Comment:
so... do you.... like... put... or... insert..?

Reply:
Ok so this time I actually added my changes :angel: